internal/database/migrations/000001_create_entities_table.up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE positions (
+                           id VARCHAR PRIMARY KEY,
+                           name VARCHAR(255) NOT NULL,
+                           salary INT NOT NULL,
+                           created_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
+                           updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP
```

Comment:
updated_at might be better to be nullable, since it'll only be updated when the record changes occurred